src/api/bannerAdmin.ts

```diff
@@ -0,0 +1,16 @@
+import firebase from "firebase/app";
+
+export const makeUpdateBanner = (
+  venueId: string,
+  onError: (errorMsg: string) => void
+) => (message?: string) => {
+  const params = {
+    venueId,
+    bannerMessage: message ? message : "",
+  };
+
+  firebase
```

Comment:
Is this intentionally without async/await?

Reply:
I don't think so. I would assume I didn't necessarily think about it because it's not returning a value. But it would still be good to expose the fact that it's technically a `Promise`

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1080/commits/7d64d0f166ee714ac276a4c885ed2c8b1af69c15